std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -111,34 +111,38 @@ struct KRRegion(ParentAllocator = NullAllocator)
 
         this(this) @disable;
 
+        nothrow @nogc @safe
         void[] payload() inout
         {
-            return (cast(ubyte*) &this)[0 .. size];
+            return (() @trusted => (cast(ubyte*) &this)[0 .. size])();
         }
 
+        nothrow @nogc @safe
         bool adjacent(in Node* right) const
         {
             assert(right);
             auto p = payload;
-            return p.ptr < right && right < p.ptr + p.length + Node.sizeof;
+            return p.ptr < right && right < (() @trusted => (p.ptr + p.length + Node.sizeof))();
```

Comment:
Same thing here. Make the whole function `@trusted`, it's too small to create a lambda.